ci/unit.yml

```diff
@@ -6,7 +6,7 @@ image_resource:
   type: docker-image
   source:
     repository: golang
-    tag: 1.19.2
+    tag: 1.22.2-bullseye
```

Comment:
Shouldn't need `bullseye` for libraries as it doesn't get deployed in a ubuntu container like apps do. It doesn't harm, but it's not necessary.

Reply:
unit build 25 begs to differ

Reply:
For completeness in PR, this was due to unsupported version of binary in downloaded package. Now fixed. Comment resolved.